### PR TITLE
docs: added information related to github action

### PR DESCRIPTION
**Pull request by ryanadhi**
Closes https://github.com/kawalcovid19/wargabantuwarga.com/issues/736

## Description

Adding recommendation to disable GitHub actions based on [this](https://github.community/t/stop-github-actions-running-on-a-fork/17965)

## Current Tasks

- [x] EN Contributing
- [x] ID Contributing

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -102,6 +102,15 @@ Closes #318
 Update **`Start working on Issues`** section with clearer instructions on getting ready to work on an issue.
 ```
 
+### GitHub Action
+
```
Comment (apraditya) [inline CONTRIBUTING.md:106]: Please update the TOC for this section.

**Review by apraditya — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ryanadhi) [reply CONTRIBUTING.md]: okay. just pushed another [commit](https://github.com/kawalcovid19/wargabantuwarga.com/pull/742/commits/08d51d96f01531e5c6ce7b59649f400fa8259147)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -102,6 +103,15 @@ Closes #318
 Update **`Start working on Issues`** section with clearer instructions on getting ready to work on an issue.
 ```
 
+### Disabling GitHub Action
```
Comment (zainfathoni) [inline CONTRIBUTING.md:106]: Please use `GitHub Actions` consistently across the documents.
```suggestion
### Disabling GitHub Actions
```

**Review by zainfathoni — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (ryanadhi) [reply CONTRIBUTING.md]: I just push your suggestion. thanks

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 

Thanks for working on it, @ryanadhi! 🙏

**Comment by zainfathoni** [comment]
@all-contributors please add @ryanadhi for doc

**Pull request merged**